torch/csrc/jit/passes/create_autodiff_subgraphs.cpp

```diff
@@ -200,37 +288,36 @@ class SubgraphSlicer {
 
   // Try to merge `producer` into `consumer`. If successful, this destroys
   // `producer` and returns the `consumer` group.
-  c10::optional<Node*> tryMerge(
-      Node* consumer,
-      Node* producer,
-      AliasDb& aliasDb) {
+  c10::optional<Node*> tryMerge(Node* consumer, Node* producer) {
     AT_ASSERT(consumer->kind() == prim::DifferentiableGraph);
     bool canMerge = shouldConsiderForMerge(producer) &&
-        aliasDb.moveBeforeTopologicallyValid(producer, consumer);
+        aliasDb_.moveBeforeTopologicallyValid(producer, consumer);
 
     if (!canMerge) {
       return c10::nullopt;
     }
 
+    ValueMapper vm(producer, aliasDb_, consumer->outputs().size());
     SubgraphUtils::mergeNodeIntoSubgraph(producer, consumer);
+    vm.copyAliasing(consumer, aliasDb_);
```

Comment:
nitpick: I wonder if it makes sense to do `copyAliasing` when vm goes out of scope?

Reply:
What do you mean ?